css/style.css

```diff
@@ -541,3 +543,75 @@ nav {
     width: 100%;
   }
 }
+
+.modal-container {
+  display: none;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.6);
+  inset: 0 0;
+  width: 100vw;
+  height: 100vh;
+  padding: 10% 0;
+  margin: 0;
+}
```

Comment:
- [ ] If the screen is smaller than the model, The complete model is not visible on the screen and your model is not scrollable so users can not see the complete model. Please use `overflow: scroll;` in your model controller so users can scroll the model.